demo/rube_testbed/loadrube.js

```diff
@@ -37,7 +37,11 @@ function loadBodyFromRUBE(bodyJso, world) {
     else
         bd.set_position(new Box2D.b2Vec2(0,0));
 
-    bd.set_gravityScale(bodyJso.gravityScale || 1);
+    if (bodyJso['gravityScale'] != undefined) {
```

Comment:
Add type check for gravityScale property https://github.com/dmagunov/box2d.js/commit/585aef75288222b35c90a3d1608b34f99d9c1cb7